src/board_list.jsx

```diff
@@ -36,9 +38,11 @@ var BoardList = React.createClass({
       );
     }
     var thumbNodes = this.props.data.map(function(data, index) {
+      let lazyLoad = (! initialLoad) || (index > 10);
+      lazyLoad = true; //test lazy loading for all images
```

Comment:
This looks like it should have been cleaned up. Let's enable lazyload for all images though.